src/hpc/autoscale/allocate.conf

```diff
@@ -0,0 +1,3 @@
+[overallocate]
```

Comment:
Can we rename this file autoscale.conf? allocate is just part of how we'd configure it.